package.json

```diff
@@ -171,6 +173,7 @@
 		"test-unit:coverage-ci": "npm run test-unit -- --coverage --maxWorkers 1 && codecov",
 		"test-unit:watch": "npm run test-unit -- --watch",
 		"test-unit-php": "docker-compose run --rm wordpress_phpunit phpunit",
-		"test-unit-php-multisite": "docker-compose run -e WP_MULTISITE=1 --rm wordpress_phpunit phpunit"
+		"test-unit-php-multisite": "docker-compose run -e WP_MULTISITE=1 --rm wordpress_phpunit phpunit",
+		"docs:generate-data-reference": "node docs/data/tool"
```

Comment:
Can you move this above, under the `dev:` commands? The rest of these commands are sorted alphabetically so that'll make them easier to scan.

Reply:
Thanks, I like to have things sorted alphabetically. It's much easier to find what you need.